NetKAN/TweakScaleCompanion.netkan

```diff
@@ -0,0 +1,31 @@
+spec_version: v1.8
+identifier: TweakScaleCompanion
+name: TweakScale Companion
```

Comment:
```suggestion
```

The `name` will be retrieved from SpaceDock automatically

Reply:
Removing data that could be retrieved from SpaceDock automatically